cypress/cypress.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'cypress'
```

Comment:
Cypress is pretty heavy weight, maybe Playwright is a better/lighter option for this project to test the basics?

Reply:
I haven't used it in the past, but I can give it a try

Reply:
@PatrickHeneise I will remove the cypress setup on this pr and raise a following one usng `Playwright` or `Cypress`. It will depend on the spike output